serving/src/main/java/ai/djl/serving/workflow/WorkflowDefinition.java

```diff
@@ -93,16 +96,21 @@ public static WorkflowDefinition parse(Path path) throws IOException {
     /**
      * Parses a new {@link WorkflowDefinition} from an input stream.
      *
-     * @param name the workflow name
+     * @param name the workflow name (null for no name)
      * @param uri the uri of the file
      * @return the parsed {@link WorkflowDefinition}
      * @throws IOException if read from uri failed
      */
     public static WorkflowDefinition parse(String name, URI uri) throws IOException {
+        return parse(name, uri, null);
+    }
+
+    static WorkflowDefinition parse(String name, URI uri, Map<String, String> templateReplacements)
```

Comment:
Only the simplified ones are public. This one and the one below (which are called from the simplified ones) are not public. I left this one as package private to make the template system go through the `WorkflowTemplates` class rather than this one